client/src/CodeChatEditor.mts

```diff
@@ -33,10 +33,20 @@ import "./graphviz-webcomponent-setup.mts";
 import "./graphviz-webcomponent/index.min.mjs";
 import { html_beautify } from "js-beautify";
 import { tinymce, tinymce_init } from "./tinymce-webpack.mjs";
+import TurndownService from "turndown";
+const turndownPluginGfm = require('turndown-plugin-gfm')
 
 // Not exactly an import, but this seems like the place to instantiate this
-const TurndownService = require("turndown");
 const turndownService = new TurndownService();
+const tables = turndownPluginGfm.tables;
+const taskListItems = turndownPluginGfm.taskListItems;
+turndownService.use([tables, taskListItems]);
+turndownService.addRule('strikethrough', {
```

Comment:
Explain this -- it's a bug fix/workaround. Is there an issue for it on the Turndown repo?

Reply:
Added more documentation for this section in [d5b9f6a](https://github.com/bjones1/CodeChat_Editor/pull/16/commits/d5b9f6a1a91911350f35043874afcdbedc3f43e4)